reactioncommerce/reaction
# Fix: console error when typing a tag name on PDP

Resolves #4438 
Impact: **minor**  
Type: **bugfix**

## Issue
On the PDP, when typing a tag name in the admin sidebar, a console error would appear: `Uncaught (in promise) TypeError: Cannot read property 'getShopId' of undefined at getShopLang (helpers.js:118)`. This was because `Reaction` was undefined when `getShopLang()` would run.

## Solution
The solution was changing a client-side `require` statement to a simple dynamic import.

## Breaking changes
None

## Testing
1. Visit PDP as admin
2. Click edit icon next to tags
3. Start typing a new tag name, confirm no console errors
4. Edit tag nav, start typing an existing tag name, confirm suggestions still work
5. Admin -> Accounts -> New group - Create a new group and confirm in the DB it has a correctly-formatted-slug
